src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -73,10 +73,20 @@ const useFagsakApi = (
             return;
         }
 
+        if (context.begrunnelse.valideringsstatus === Valideringsstatus.IKKE_VALIDERT) {
```

Comment:
Remove this(since we have it in the reducer). I think we should move the form validation to the useFagsak file. Because now the validation will not run if the user does not click the begrunnelse field.

Reply:
Why? This is just for secure validation when user click next without typing anything

Reply:
Much like we do here: https://github.com/navikt/familie-ba-sak-frontend/blob/master/src/frontend/komponenter/Fagsak/Beregning/BeregningProvider.tsx#L60